ansible/README.md

```
@@ -0,0 +1,262 @@
+Deploys everything from scratch:
+```bash
+./scripts/deploy.sh -e production
+```
+
+### Quick Updates
+For code updates without full infrastructure changes:
+```bash
+./scripts/deploy.sh -u
+```
+
+### Targeted Deployment
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)